Tests/SentryTests/Integrations/ViewHierarchy/TestSentryViewHierarchyProvider.swift

```diff
@@ -19,14 +21,14 @@ class TestSentryViewHierarchyProvider: SentryViewHierarchyProvider {
         saveFilePathUsed = filePath
         return true
     }
+}
 
-    override func viewHierarchy(from view: UIView!, into context: UnsafeMutablePointer<SentryCrashJSONEncodeContext>!) -> Int32 {
-        return viewHierarchyResult != 0 ? viewHierarchyResult : super.viewHierarchy(from: view, into: context)
-    }
+class TestSentryViewHierarchyProviderHelper: SentryViewHierarchyProviderHelper {
+    
+    static var viewHierarchyResult: Int32 = 0
```

Comment:
`l`: Can be simplified using nullable value

```suggestion
    static var viewHierarchyResult: Int32?
```